deploy/vagrant/stop

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+HERE=`dirname $0`
+pushd $HERE >/dev/null
+vagrant destroy -f
+popd >/dev/null
```

Comment:
`start` and `stop` kind of imply that the cluster will be reused? Maybe `create` and `destroy`?